app/elements/candidates/invite-candidates.html

```diff
@@ -78,14 +74,13 @@
 				}
 			},
 			_addCandidate: function(){
-				if (this.challenge){
-					this.push('candidates', {'Challenge': this.challenge.Key});
+				if (this.challenges.length == 1){
+					this.push('candidates', {'Challenge': this.challenges[0].Key});
```

Comment:
You can input this by default even if `this.challenges.length > 1` because the user will see the first challenge selected and if he invites without modifying the challenge the user will not have a challenge set.